src/library_webgpu.js

```diff
@@ -40,18 +40,18 @@
       return s;
     },
 
-    makeU64ToNumber: function(lowName, highName) {
+    makeI32I32ToU53: function(lowName, highName) {
```

Comment:
We have a bunch of int53 utility functions in `library_int53.py`.. can we unify these maybe?  Perhaps as a followup?